src/entity/statuses.ts

```diff
@@ -60,6 +60,9 @@ export class StatusEntity {
   @Column({ default: 18 })
   public maxZoom: number;
 
+  @Column()
+  public sourceLayer: string;
```

Comment:
add to open api schema of "statusRequest"

Reply:
Add also to the post data in order to get it from outside

Reply:
fixed in eaf10b1